config/rewrites.d/support.rackspace.com.json

```diff
@@ -885,7 +885,7 @@
       {
         "description": "Retire Administrator management: Cloud Office Control Panel",
         "from": "^\/how-to\/update-billing-information-for-cloud-hosting(.*)$",
```

Comment:
This line should resemble: `"from": "^\\/how-to\\/set-an-email-catch-all-address-in-the-cloud-office-control-panel(.*)$",`